notebooks/Time_Series_Forecasting.ipynb

```diff
@@ -0,0 +1,660 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [],
+   "source": [
+    "stores_data = pd.read_csv('C:\\\\Users\\\\Owner\\\\Desktop\\\\Proga\\\\REPOS\\\\mentorship_EPAM\\\\data\\\\stores.csv')\n",
+    "stores_data.head()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "stores_data.info()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<i>Store id doesn't help itself with the sales predictions, so it is necessary to replace the store id with the corresponding information about it</i>"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "train_data = train_data.merge(stores_data, on=\"store_nbr\", how=\"left\")\n",
+    "train_data = train_data.drop('store_nbr', axis=1)\n",
+    "train_data.head()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "print('Number of \\'city\\', \\'state\\', \\'type\\' unique values: \\n')\n",
+    "for feature in ['city', 'state', 'type']:\n",
+    "    print(feature, len(train_data[feature].unique()))"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "plt.figure(figsize=(25,9))\n",
+    "sns.countplot(x=train_data['city'], alpha=0.7, data=train_data)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "plt.figure(figsize=(25,9))\n",
+    "sns.countplot(x=train_data['state'], alpha=0.7, data=train_data)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<i>It seems that the presence of both features 'state' and 'city' isn't necessary, that is why we should delete one of them.<br>I'll choose 'state' feature, because 'city' feature gives us more information (there can be few cities in the state).<br>'city' feature has only 7 more values than 'state' feature, that is why speaking about the model complexity, there should not be much difference.</i>"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "train_data = train_data.drop('state', axis=1)\n",
+    "train_data.head()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "train_data_copy = train_data.copy()\n",
+    "train_data_copy.head()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "plt.figure(figsize=(20, 100))\n",
```

Comment:
As already discussed some plots suffer from Y-axis scaling.
Can you make any conclusions on these plots?